splitio/redis_support.py

```diff
@@ -66,21 +66,25 @@ def register_segment(self, segment_name):
         :param segment_name: Name of the segment.
         :type segment_name: str
         '''
-        self._redis.sadd(
-            RedisSegmentCache._KEY_TEMPLATE.format(suffix='registered'),
-            segment_name
-        )
+        # self._redis.sadd(
```

Comment:
If everything works properly and QOS is green, we should remember to remove these lines before the final merge to master/release to pypi and just leave a comment `# NOTE: segments needn't be registered/unregistered by the sdk when running in redis consumer mode. These operations are handled by the split-synchronizer (link to the split-sync docs)`

Reply:
sounds good